leaflet/index.d.ts

```diff
@@ -1116,19 +1120,19 @@ declare namespace L {
         removeHooks(): void;
     }
 
-    export interface Event {
+    export class Event {
```

Comment:
I'm not sure if `Event` and any of its children need to be classes. I've been looking for evidence of it on the source code but I can't find any declarations that would suggest they are classes, unless they're autogenerated. Are you certain they can be `new`'ed?